mappings/net/minecraft/state/property/Properties.mapping

```diff
@@ -83,3 +83,7 @@ CLASS bwb net/minecraft/state/property/Properties
 	FIELD x SHORT Lbwc;
 	FIELD y SIGNAL_FIRE Lbwc;
 	FIELD z SNOWY Lbwc;
+	METHOD a (Lbwo;)Z
```

Comment:
what does this boolean return value mean?

Reply:
It is a synthetic method for a predicate.